### ISSUE TITLE
Shorting these two open pads is easier

### ISSUE THREAD

**Issue by nelsonjchen** [nelsonjchen/c3-faux-touch-keyboard#12]
https://github.com/eccherda/ch552g_mini_keyboard/blob/master/readme.md#setting-up-the-keyboard-in-bootloader-mode

**Comment by nelsonjchen** [comment]
These pads can be shorted with the screws from the keyboard!

**Comment by Kaofui** [comment]
![image0](https://github.com/nelsonjchen/c3-faux-touch-keyboard/assets/138717561/4e150aee-24b9-4a6e-9026-39d080766d5d)

**Comment by nelsonjchen** [comment]
@eccherda 

Thanks for the writing your [bootloader instructions with the R12 pad being usable to get the keyboard into bootloader mode.](https://github.com/eccherda/ch552g_mini_keyboard/blob/master/readme.md#setting-up-the-keyboard-in-bootloader-mode).

I rewrote my instructions for this keyboard firmware to use that instead of trying to connect two pins together. In fact, I also rewrote it to use [the pads with the screw head from the bottom of the screwdriver as a shorting tool while applying pressure downward with the keyboard on a soft surface since the screws are conductive so extra hands or hardware isn't needed](https://github.com/nelsonjchen/c3-faux-touch-keyboard?tab=readme-ov-file#getting-the-keyboard-into-bootloader-mode).
